lib/api/2.0/notification.js

```diff
@@ -30,7 +36,124 @@ var notificationProgressPost = controller(function(req, res){
     });
 });
 
+/**
+ * @api {post} /api/2.0/notification/alerts
+ * @apiDescription Receives Redfish alerts, refactors the message and publishs it into amqp
+ * @apiName notification post
+ * @apiGroup notification
+ */
+
+var notificationAlertsPost = controller({success: 201}, function(req,res) {
```

Comment:
Here we go: 
`{ '{"Context":"context string","EventId":"8491","EventTimestamp":"2017-03-30T11:27:15-0500","EventType":"Alert","MemberId":"7e675c8e-127a-11e7-9fc8-64006ac35232","Message":"Successfully logged in using root, from 10.252.79.21 and GUI.","MessageArgs":["root","10.252.79.21","GUI"],"MessageArgs@odata.count":3,"MessageId":"USR0030","OriginOfCondition":"iDRAC.Embedded.1","Severity":"Informational"}\n': '' }`